src/components/Filter/components/FilterSectionHeader/filterSectionHeaderUtil.tsx

```diff
@@ -1,6 +1,6 @@
 import { FilterLabels, FilterValueChip } from './filterSectionHeaderStyles';
 import { FilterModule } from 'src/components/Filter/types';
-import { ReactComponent as CloseIcon } from 'src/components/Modal/assets/closeIcon.svg';
+import { ReactComponent as CloseIcon } from '../../../Modal/assets/closeIcon.svg';
```

Comment:
could we return this to be relative to root instead of using `../../../`?

Reply:
For the svgs we have to import them with the relative path: https://github.com/ToyotaResearchInstitute/lakefront/blob/main/CONTRIBUTING.md#using-svgs

This import was from my filter story. I didn't realize using 'src/components...' for the svg would not link the correct path. I wanted to reuse an svg that we already had within the application without recreating another copy.